src/Invoice_GenUI/Invoice_GenUI/ViewModels/ShowClientsViewModel.cs

```diff
@@ -39,8 +40,12 @@ private void GoBack()
             Navigation.NavigateTo<HomeViewModel>();
         }
         [RelayCommand]
-        private void ClientDetails()
+        private void ClientDetails(object parameter)
         {
+            if (parameter is CreateClientModel client)
```

Comment:
❓ genuine question: what is `client` isn't a CreateClientModel? Does the parameter _have_ to be an `object` or can the method be written such that the parameter is a `CreateClientModel`?